### ISSUE TITLE
Idna compliant

### ISSUE THREAD

**Issue by spirillen** [piperun/iploggerfilter#49]
You should convert sunrise domains into idna valid domains for broader support, `rëddït.com = xn--rddt-lpa0a.com`

https://github.com/piperun/iploggerfilter/blob/820a540a97352911d88ca94bcc1bdc23b0a78699/filterlist#L66

**Comment by piperun** [comment]
Thanks for the issue! I'll look into it when I can (inactive due to injury).

**Comment by spirillen** [comment]
Let's hope it ain't your head... I have enough of that myself for several good people.

I can make it rather ASAP with PyFunceble if you like, and the output in the files of https://github.com/piperun/iploggerfilter/issues/50 should already be IDNA formatted, Converted.

Answer to: https://github.com/piperun/iploggerfilter/issues/50

If you give me a should head up of what your idea is, I can make a PR on this, less work to you.

**Comment by piperun** [comment]
It thankfully isn't! Hope your head also recovers well! 😅

#49
I would say create a new category (#) which is named something like:
`# Idna compliant`
and then the subcategory:
`#blazse.com`

#50
The general rule is to keep inactive domains since unless the domain owner has shown that the domain isn't owned by said ip logging company and that the current domain is used for "good" then it gets removed.

Happy new year!

**Comment by spirillen** [comment]
Is this somewhat you had in mind https://github.com/dns-test/iploggerfilter ??

**Comment by piperun** [comment]
Yes something like that, however when I compare yours with mine I'm noticing quite a lot of domains missing:
```zzssfc.tk
tompriki.gr
tokdigitalexpress.com
```
to name a few.

**Comment by spirillen** [comment]
They are probably in the inactive lists :wink: 

```
git grep 'zzssfc.tk'
Test_results/inactive.txt:||zzssfc.tk^$important,all
```

**Comment by piperun** [comment]
I see, well for the sake of consistency it's better to have it be part of the main filterlist, since as long as the domain is simply inactive/invalid and hasn't changed owner it's better to be safe than sorry, if the current domain owners were to use the domain again.

**Comment by spirillen** [comment]
I can do that,but then the hole idea behind taking the dead one away gone, the dead once in this results is DEAD on all levels, the only way they in 99% of the cases to reactivate these domains is by reregister them.

The test is set. so if one test reply an active, the result is active, it is a rather trustworthy result, the reason I left them in a separate file is, nothing is fail proof, and as you set, if they come back online. So next time you test them you just run the test on both file, if anything become a live you just reactive the rule :smile:  In this way you should consider the rule de-activated, not deleted.

```
whois zzssfc.tk
The domain you requested is not known in Freenoms database.
```
